.github/workflows/integration_tests.yml

```diff
@@ -2,7 +2,7 @@ name: Integration Tests
 on:
   schedule:
     # * is a special character in YAML so you have to quote this string
-    - cron: '0 1 * * *' # run integration tests at 1 AM (UTC), monday to friday (1-5)
```

Comment:
I would only change this and left the extra Debug level logs :) They can be handy in future

Reply:
adapted the logs and kept them :)